source/intro_to/explore_software.rst

```diff
@@ -0,0 +1,229 @@
+
+:bash:`topup` is a tool that corrects for susceptibility-induced distortions in diffusion MRI data.
+
+:bash:`xtract` (cross-species tractography) is a tool that automatically extract a set of tracts in humans and macaques. It can also be used to define one's own tractography protocols where all the user needs to do is to define a set of masks in standard space (e.g. MNI152).
+
+**TBSS** is Tract-Based Spatial Statistics, part of FMRIB's Diffusion Toolbox, which offers voxelwise analysis of multi-subject diffusion data.
+
+Installation
+^^^^^^^^^^^^^^^^^^^^^
+
+Visit this website for download and install instructions as well as a general overview of tools available in the package `(here) <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki>`__
+
+You will have to download a Python file install (after accepting a license) and then install FSL (by executing the Python installer)
+`(here) <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation>`__
+
+It is common to add these lines to your :bash:`.bashrc` to make FSL easier to use;
+
+.. code-block:: bash
+
+    # export FSLDIR=/PATH/TO/FSL
+    . ${FSLDIR}/etc/fslconf/fsl.sh
+    export PATH=${FSLDIR}/bin:${PATH}
+
+Once installed, you should be able to type :bash:`bet` or :bash:`fast` to see the help display. :bash:`fsleyes` will launch the visualization tool.
+
+
+ANTs
+-----------------------
+ANTs (Advanced Normalization Tools) is a powerful open-source software package for (medical) image analysis registration and segmentation. Extremely useful to create templates or extract cortical thickness. It is widely used in the field of neuroimaging.
+You can find information on their `Github page <https://github.com/ANTsX/ANTs>`__ or on their new `documentation <http://stnava.github.io/ANTsDoc/>`__.
+
+Useful Commands
+^^^^^^^^^^^^^^^^^^^^^
+
+:bash:`antsRegistrationSyN.sh` To compute registration (both linear and nonlinear). The main use is to bring any subject to MNI space or to co-registered different modalities of a single subject. Be careful, this version of the script use the correlation coefficient (CC) as a metric, meaning only correlated image intensites will work (T1w to T1w, T1w to FLAIR, or b0 to FMRI). This script runs in 10-20 minutes.
+
+:bash:`antsRegistrationSyNQuick.sh` To compute registration (both linear and nonlinear). This version of the script use the Mutual Information (MI) as a metric, meaning statistical dependencies between image intensites will work (T1w to b0, FLAIR to PET, or T2w to CT). This script is much faster and is often use for  to bring the T1w image into the diffusion space or for a quick registration (1 minute, lower quality) between image.
+
+:bash:`antsApplyTransforms` To apply any transformations from ANTs to other images. Useful after registration of two images to transform masks and labels. Be sure to apply the transformation in the right order and use the right datatype (ask Francois Rheault about it).
```

Comment:
Useful after registration of two images to transform masks and labels.

==>

Useful after registration of two images to apply the same transformation to other images that are in the same space, such as masks and labels associated to your moving image.